GliaWidgets/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -42,6 +43,7 @@ class ChatViewModel: EngagementViewModel, ViewModel {
             declined: () -> Void
         )
         case showCallBubble(imageUrl: String?)
+        case updateNewMessageIndicator(itemCount: Int)
```

Comment:
Would rename to `updateUnreadMessageCount(Int)`

Reply:
`unreadMessages` refers not only to the new message indicator though but also to a minimized call bubble (if you receive a message while a call is minimized, a small badge with a number is shown). This specific `action` is aimed towards a view specifically and only updates the new message indicator.

Maybe `updateNewMessageIndicator(count: Int)` or `updateNewMessageIndicatorCount(Int)` instead?

Reply:
UPD: taking into consideration the renaming of the entire thing into an `unreadMessageIndicator` I suggest

`updateUnreadMessageIndicator(count: Int)` or `updateUnreadMessageIndicatorCount(Int)`, because it still only affects the view and not the unread messages observer / call bubble.